From 5c249e7d71f50691d38c27316514f2f2882c50f8 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Wed, 15 May 2013 10:29:35 +0200
Subject: [PATCH 242/628] spi: xilinx-qps: Fix coding style issues

git://github.com/Xilinx/linux-xlnx.git xilinx-v14.7
commit 5ead10be8a1c4c67e6de5007419acd28502ee1eb

Fix some checkpatch.pl warnings.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Liming Wang <liming.wang@windriver.com>
---
 drivers/spi/spi-xilinx-qps.c |    9 ++++-----
 1 files changed, 4 insertions(+), 5 deletions(-)

diff --git a/drivers/spi/spi-xilinx-qps.c b/drivers/spi/spi-xilinx-qps.c
index 8883524..872420a 100644
--- a/drivers/spi/spi-xilinx-qps.c
+++ b/drivers/spi/spi-xilinx-qps.c
@@ -153,7 +153,7 @@
  * @speed_hz:		Current QSPI bus clock speed in Hz
  * @trans_queue_lock:	Lock used for accessing transfer queue
  * @config_reg_lock:	Lock used for accessing configuration register
- * @txbuf: 		Pointer	to the TX buffer
+ * @txbuf:		Pointer	to the TX buffer
  * @rxbuf:		Pointer to the RX buffer
  * @bytes_to_transfer:	Number of bytes left to transfer
  * @bytes_to_receive:	Number of bytes left to receive
@@ -229,7 +229,7 @@ static struct xqspips_inst_format flash_inst[] = {
  * The default settings of the QSPI controller's configurable parameters on
  * reset are
  *	- Master mode
- * 	- Baud rate divisor is set to 2
+ *	- Baud rate divisor is set to 2
  *	- Threshold value for TX FIFO not full interrupt is set to 1
  *	- Flash memory interface mode enabled
  *	- Size of the word to be transferred as 8 bit
@@ -428,9 +428,8 @@ static int xqspips_setup_transfer(struct spi_device *qspi,
 		return -EINVAL;
 	}
 
-	if (transfer && (transfer->speed_hz == 0)) {
+	if (transfer && (transfer->speed_hz == 0))
 		req_hz = qspi->max_speed_hz;
-	}
 
 	/* Set the clock frequency */
 	if (xqspi->speed_hz != req_hz) {
@@ -646,7 +645,7 @@ static int xqspips_start_transfer(struct spi_device *qspi,
 		     (instruction == XQSPIPS_FLASH_OPCODE_DUAL_READ) ||
 		     (instruction == XQSPIPS_FLASH_OPCODE_QUAD_READ))) {
 
-			u8 *ptr = (u8*) (xqspi->txbuf);
+			u8 *ptr = (u8 *) (xqspi->txbuf);
 			data = ((u32) ptr[1] << 24) | ((u32) ptr[2] << 16) |
 				((u32) ptr[3] << 8) | ((u32) ptr[4]);
 			data = data >> 1;
-- 
1.7.5.4

